09-Helm.md

```diff
@@ -15,62 +15,57 @@ Helm is a “package manager” for Kuberentes. It allows a simple, single comma
 - Release: The instance of the created application. Each `helm upgrade` or `helm install` command creates a new release
 - Values: The values used to configure the templated yaml. Most charts have sensible defaults and a few required values. Values can be passed in on the command line or through a yaml file(s)
 - Helm: The local command
-- Tiller: The in-cluster server component
 
 **Preparation:**
 
-1.  Setup Kubernetes using Kind, Minikube, k3s.io, or EKS:
+1. Setup Kubernetes using Kind or Minikube:
```

Comment:
Limiting options as I tried to install Kind unsuccessfully into my docker-machine. Minikube worked ok though.